continuous-testing/use-cases/markmap.md

```diff
@@ -1,56 +1,134 @@
 * Continuous Testing
     * For Development APIs
-        * Unit Testing
-            * xUnit (e.g. Junit\[Java\], PyUnit\[Python\], NUnit\[C#\], CppUnit\[C++\]
-            * Tools
-                * Java
-                * Python
-                * C#
-                * C/C++
-        * Integration, Dependency and Stub Mocking
-            * Simulate dependencies
-            * Stub out coupled services for planned results
-            * Mock specific application behavior
-            * Tools
-                * Java
+        * For unit testing, integration, dependency and stub mocking
+            * For module testing
+                * Using xUnit
+                    * (Java) JUnit [&#x1F3C1; STARTER KIT](../starter-kits/#unit-teting-with-java)
+                    * (Python) PyUnit [&#x1F3C1; STARTER KIT](../starter-kits/#unit-teting-with-python)
+                    * (C#) NUnit
```

Comment:
This applies for all tools listed, but please add hyper-links to all tools' homepages, or documentation.

Reply:
Done